### ISSUE TITLE
Multi-level navigation support

### ISSUE THREAD

**Issue by hylerrix** [xcatliu/pagic#29]
**Is your feature request related to a problem? Please describe.**

Pagic now supports Level 1 navigation, But I want Secondary navigation support.

**Describe the solution you'd like**

Something like this...?:

```
sidebar: [
    'README.md',
    {
      link: 'articles/basic/README.md',
      children: [
        {
          link: 'articles/basic/chapter01/README.md',
          children: [
            'articles/basic/chapter01/install-and-hello-world.md',
          ]
        }
      ]
    },
]
```

**Comment by hylerrix** [comment]
And another feature which I want is `pagic.config.tx` hot-reload support.

Now, when I change the `pagic.config.tx` source code, terminal says `[Pagic] Copy ...pagic.config.ts` but must re-run `deno run` script to refresh.

**Comment by xcatliu** [comment]
Good suggestion. Will support these in the next version

**Comment by xcatliu** [comment]
Published in version v0.8.0🎉

Support multi-level sidebar and watch `pagic.config.ts` changes
